distroless.unknown.X9 copy.Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG IMAGE
```

Comment:
`distroless.unknown.X9 copy.Dockerfile` ? copy?

Reply:
OMG the ctrl+c/v curse. I will fix.